ArduPlane/mode.h

```diff
@@ -0,0 +1,475 @@
+#pragma once
+
+#include <stdint.h>
+
+#include <GCS_MAVLink/GCS_MAVLink.h>
+#include <AP_Math/AP_Math.h>
+
+#include "defines.h"
+
+#define MODE_NEXT_HEADING_UNKNOWN   99999.0f    // used to indicate to set_desired_location method that next leg's heading is unknown
```

Comment:
This is definitely not required.

Reply:
fixed. I removed a bunch of the copy/paste leftover stuff at the top